config/initializers/devise.rb

```diff
@@ -253,7 +253,10 @@
   # Add a new OmniAuth provider. Check the wiki for more information on setting
   # up on your models and hooks.
   # config.omniauth :github, 'APP_ID', 'APP_SECRET', scope: 'user,public_repo'
-
+  config.omniauth :cas, host: 'login.oregonstate.edu', url: 'https://login.oregonstate.edu/cas'
+  config.omniauth :saml,
+    idp_cert: 'cert',
```

Comment:
Also, since we don't have a final URL for production yet, I can only submit for trusted access for staging. If URLs change, I'm pretty sure our trusted status breaks.

Reply:
Good idea to move these strings to ENV variables @straleyb with defaults..